common/script_values/imperia_values.txt

```diff
@@ -1773,3 +1773,76 @@ imperia_china_infamy_penalty_value = {
 		}
 	}
 }
+
+imperia_money_transfer_ten_million_counter = {
+	value = 0
+	add = {
+		value = var:imperia_money_transfer_amount
+		divide = 10000000
+		#modulo = 10
+		floor = 1
+	}
+}
+
+imperia_money_transfer_one_million_counter = {
+	value = 0
+	add = {
+		value = var:imperia_money_transfer_amount
+		divide = 1000000
+		modulo = 10
+		floor = 1
+	}
+}
+
+imperia_money_transfer_one_hundred_k_counter = {
+	value = 0
+	add = {
+		value = var:imperia_money_transfer_amount
+		divide = 100000
+		modulo = 10
+		floor = 1
+	}
+}
+
+imperia_money_transfer_ten_k_counter = {
+	value = 0
+	add = {
+		value = var:imperia_money_transfer_amount
+		divide = 10000
+		modulo = 10
+		floor = 1
+	}
+}
+
+imperia_money_transfer_one_k_counter = {
+	value = 0
+	add = {
+		value = var:imperia_money_transfer_amount
+		divide = 1000
+		modulo = 10
+		floor = 1
+	}
+}
+
+imperia_remaining_credits_value = {
+	value = credit
+	subtract = imperia_current_debt_value
+}
+
+imperia_current_debt_value = {
+	value = credit
+	multiply = scaled_debt
+}
+
+imperia_current_transferable_money_value = {
+	value = 0
```

Comment:
The value = 0 doesn't do anything you can just do value = gold_reserves like the ones above.